admin/pycodestyle-twisted.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
```

Comment:
This is missing a copyright notice. Also shouldn't we reproduce the pycodestyle license https://github.com/PyCQA/pycodestyle/blob/master/LICENSE here in this file as `blank_lines()` was partially copied from that repository? Not that we are distributing this script in our releases, but best to overcommunicate on these legal things.

Reply:
done. Let me know if this is ok... otherwise feel free to update and push your suggested changes :)

Reply:
I can do that :)